lib/signed-xml.js

```diff
@@ -18,9 +18,16 @@ function FileKeyInfo(file) {
   this.file = file
 
   this.getKeyInfo = function(key, prefix) {
-    prefix = prefix || ''
-    prefix = prefix ? prefix + ':' : prefix
-    return "<" + prefix + "X509Data></" + prefix + "X509Data>"
+    let currentPrefix = prefix || ''
+    currentPrefix = currentPrefix ? currentPrefix + ':' : currentPrefix
+    let signingCert = '';
+    if (key) {
+      let certArray = Array.isArray(key) ? key : [key];
+      for(let cert in certArray) {
+        signingCert += "<" + currentPrefix + "X509Certificate>" + cert + "</" + currentPrefix + "X509Certificate>"
+      }
+    }
+    return "<" + currentPrefix + "X509Data>" + signingCert + "</" + currentPrefix + "X509Data>"
```

Comment:
We're trying to maintain backward compatibility with Node 4 here, since enterprise environments might be extremely slow in moving. Could we do..
```suggestion
    var currentPrefix = prefix || ''
    currentPrefix = currentPrefix ? currentPrefix + ':' : currentPrefix
    var signingCert = '';
    if (key) {
      var certArray = [].concat(key);
      for(var i = 0; i < certArray.length; ++i) {
        var cert = certArray[i]
        signingCert += "<" + currentPrefix + "X509Certificate>" + cert + "</" + currentPrefix + "X509Certificate>"
      }
    }
    return "<" + currentPrefix + "X509Data>" + signingCert + "</" + currentPrefix + "X509Data>"
```